pkg/apis/options/secret_source.go

```diff
@@ -0,0 +1,14 @@
+package options
+
+// SecretSource references an individual secret value.
+// Only one source within the struct should be defined at any time.
+type SecretSource struct {
+	// Value expects a base64 encoded string value.
+	Value []byte `yaml:"value,omitempty"`
+
+	// FromEnv expects the name of an environment variable.
+	FromEnv string `yaml:"fromEnv,omitempty"`
```

Comment:
Not blocking for this PR, but I thought now we have env substitution, we won't need this right?